Core/Tests/Directory.Build.props

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@AlanRynne for the test projects (Unit + integration + performance)

I think we'd want something like this, with far fewer of the analysers running. 
My thought is, we often are unit testing things that (like how our serializer deals with obsolete functions, nested classes, classes etc.) all things that trigger a warning.
Let me know what you think, this is just a first pass

Reply:
OK cool. I did have this though that we may want to reuse. Glad to hear its easy to pull off!

Reply:
Implemented a solution with alan that uses the `IsTestProject` prop rather than unique props files.